projects/battery_heartbeat/inc/battery_heartbeat.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include "can_ack.h" 
+#include "exported_enums.h" 
+
+//from ms12 code, not sure if these are still current
```

Comment:
remove this comment, don't leave comments that are like "idk if this should be here or not", ask questions...